include/seqan3/search/algorithm/detail/search_scheme_precomputed.hpp

```diff
@@ -69,7 +69,7 @@ struct search
 };
 
 //!\brief Object storing information for a search (of a search scheme).
-//        Number of blocks do not have to be known at compile time.
+//!       Number of blocks do not have to be known at compile time.
```

Comment:
Is this detail of the doxygen entity?

Reply:
I'm not really sure what it is (or if it's supposed to be brief or detail), it shows as
```
Object storing information for a search (of a search scheme). Number of blocks have to be known at compile time.
```
in the docs. So I guess it's a brief?

Reply:
well that's the question. I would rather put it into an attention section?